docs/examples/vars.yaml

```diff
@@ -44,3 +44,11 @@ other:
   - name: "non-cluster-vm"
     ipaddr: "192.168.7.31"
     macaddr: "52:54:00:f4:2e:2e"
+
+chronyconfig:
+  enabled: false
+  content:
+    - server: 0.centos.pool.ntp.org
+      options: iburst
+    - server: 1.centos.pool.ntp.org
+      options: iburst
```

Comment:
Only feedback I have is that the `chronyconfig` should not be in the "default" examples. This config should be in the `vars/main.yaml` file as...

```
chronyconfig:
  enabled: false
```

The user then overrides this setting.

I'm not opposed to having a `docs/examples/vars-chrony.yaml` example though.